components/notebook-controller/controllers/notebook_controller.go

```diff
@@ -324,6 +269,102 @@ func (r *NotebookReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	return ctrl.Result{RequeueAfter: culler.GetRequeueTime()}, nil
 }
 
+func updateNotebookStatus(r *NotebookReconciler, nb *v1beta1.Notebook,
+	sts *appsv1.StatefulSet, req ctrl.Request) (bool, error) {
+
+	log := r.Log.WithValues("notebook", req.NamespacedName)
+	ctx := context.Background()
+
+	statusUpdate := false
+	// Initialize Notebook CR Status
+	if nb.Status.Conditions == nil {
```

Comment:
we should recalculate the status on each reconciliation.

So let's just do here the 
```golang
status := v1beta1.NotebookStatus{
    Conditions:     make([]v1beta1.NotebookCondition, 0),
    ReadyReplicas:  sts.Status.ReadyReplicas,
    ContainerState: corev1.ContainerState{},
}
```

and then in the following code setup accordingly the `conditions` and `containerState`

Reply:
Fixed!